Directory.Packages.props

```diff
@@ -122,14 +122,14 @@
     <PackageVersion Include="Microsoft.Extensions.Caching.Memory" Version="8.0.1" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'net9.0'" >
+  <ItemGroup Condition="'$(TargetFramework)' != 'net8.0'">
     <PackageVersion Include="Microsoft.Extensions.Caching.Memory" Version="9.0.3" />
   </ItemGroup>
 
   <!-- These versions are used for the NuGet packages that are dependent on the current TFM -->
   <!-- There may be no TFM in an evaluation phase so we can't use a conditional 'Property' -->
   
-  <ItemGroup Condition="'$(TargetFramework)' == 'net9.0'">
+  <ItemGroup Condition="'$(TargetFramework)' != 'net8.0'">
```

Comment:
```suggestion
  <ItemGroup Condition="'$(TargetFramework)' > 'net8.0'">
```

Reply:
I assume you randomly suggested that, because I am pretty sure that it won't do what we would guess. Or maybe work "by chance". The `$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net9.0'))` I use in other places might just be better off.

Reply:
![image](https://github.com/user-attachments/assets/7ca13b1f-4510-4762-b936-08064f8c7c54)

Reply:
> I assume you randomly suggested that, because I am pretty sure that it won't do what we would guess. Or maybe work "by chance".

I assume it will work for `net9.0` and `net10.0`

Reply:
It won't the screenshot I shared shows how comparison works on strings. The method I mentioned would probably though. But it doesn't matter really since when we need to change the tfms it will be only net10.0, or 9 + 10, and in that case there won't be conditions.